### PR TITLE
Registrating- Mudit Garg

### PR DESCRIPTION
**Pull request by mudit999**
Hello TeamTanay!!

You guys are really doing a great job.
Please accept my request :)

### CODE REVIEW

File: content/participants/ttjc@mudit999/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+linkedin-link: "https://www.linkedin.com/in/garg-mudit/"
+looking-for: "job"
+description: "Computer Science | UPES,Dehradun | Beginner - Learning JavaScript, React.js"
+---
+
+Hello! My name is Mudit Garg
+
+### introduction
+
+I am 3 Year B.Tech student. Pursing B.Tech from UPES, Dehradun. I am a Beginner. I have worked a little bit in HTML,CSS, JavaScript but completely new to  Node.js, React.js
```
Comment (khusharth) [inline content/participants/ttjc@mudit999/index.md:16]: Hey! @mudit999 a small mistake here:
* I am **3** year ... -> a 3rd

File: content/participants/ttjc@mudit999/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Hello! My name is Mudit Garg
+
+### introduction
+
+I am 3 Year B.Tech student. Pursing B.Tech from UPES, Dehradun. I am a Beginner. I have worked a little bit in HTML,CSS, JavaScript but completely new to  Node.js, React.js 
+
+## why are you taking this challenge?
+
+I need an Job in May-June 2021.
```
Comment (khusharth) [inline content/participants/ttjc@mudit999/index.md:20]: a* Job

File: content/participants/ttjc@mudit999/index.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+I will focus on building strong foundation of JavaScript and learning these frameworks.
+I want to work with mentor and hopefully contribute in Open Source and do GSoC
```
Comment (khusharth) [inline content/participants/ttjc@mudit999/index.md:22]: work with **a** mentor

File: content/participants/ttjc@mudit999/index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+I like to watch Marvel movies, I like photography
+
+### help I need?
+
+I will reach out if I needed any help. 
+
+### how to contact me?
+
+- {LinkedIn}(https://www.linkedin.com/in/garg-mudit/)
```
Comment (khusharth) [inline content/participants/ttjc@mudit999/index.md:34]: Please use the correct format to add links in markdown. **{ }** are wrong use square brackerts **[ ]**

File: content/participants/ttjc@mudit999/index.md (hunk 2 above)
Comment (mudit999) [reply content/participants/ttjc@mudit999/index.md]: @khusharth 
Updated my application!!
Thank you :)

**Comment by khusharth** [comment]
@gkanishk  Good to merge.

**Pull request merged**